man/dracut.cmdline.7.adoc

```diff
@@ -1414,8 +1414,6 @@ rd_NO_MD:: rd.md=0
 
 rd_MD_UUID:: rd.md.uuid
 
-rd_NO_MULTIPATH: rd.multipath=0
```

Comment:
I would expect such kind of removal in a separate commit with a reason given in the git commit messages.